app/controllers/api/v3/courses_controller.rb

```diff
@@ -24,11 +36,19 @@ def show
 
     private
 
+      # Used for sitemap on Find
+      # When these fields are specified return all results for sitemap
+      def disable_pagination?
+        if params[:fields] && params[:fields][:courses]
+          (params[:fields][:courses].split(",") & %w[course_code provider_code updated_at]).size == 3
+        end
+      end
+
       def build_courses
         @courses = if @provider.present?
                      @provider.courses
```

Comment:
Does this automatically include providers?

Reply:
It does not. I have now added this